python/tank/platform/qt/build_resources.sh

```diff
@@ -11,7 +11,7 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 # The path to where the PySide binaries are installed
-PYTHON_BASE="/Applications/Shotgun.app/Contents/Frameworks/Python"
+PYTHON_BASE="/Applications/Shotgun.app/Contents/Resources/Python"
```

Comment:
No, you are fine. We must not have recompiled the resources for core since Python was moved inside the desktop bundle.